awx/ui_next/src/screens/Inventory/InventoryList/InventoryList.jsx

```diff
@@ -188,6 +191,17 @@ function InventoryList({ i18n }) {
           ]}
           toolbarSearchableKeys={searchableKeys}
           toolbarRelatedSearchableKeys={relatedSearchableKeys}
+          headerRow={
+            <HeaderRow defaultSortKey="name" qsConfig={QS_CONFIG}>
```

Comment:
Since we will be doing this all over the place I wonder if there is some sort of hook we can use to generate the header rows for us.  I'm imagining somewhere where we send it an array or column header names and it returns the correct header cells.

Reply:
HeaderRow is a custom component to take care of a bunch of the sorting logic for us already. I could toy with passing the column data as a prop or something.

I also considered exporting the HeaderRow from the item file (e.g. export InventoryHeaderRow from InventoryListItem) so that the column data all lives in the same place instead of keeping it in sync across two files